scripts/openmc-ace-to-hdf5

```diff
@@ -124,47 +124,81 @@ for filename in ace_libraries:
     for table in lib.tables:
-        if table.name.endswith('c'):
+        name, xs = table.name.split('.')
+        if xs.endswith('c'):
             # Continuous-energy neutron data
-            try:
-                neutron = openmc.data.IncidentNeutron.from_ace(
-                    table, args.metastable)
-            except Exception as e:
-                print('Failed to convert {}: {}'.format(table.name, e))
-                continue
-
-            # Fission energy release data, if available
-            if args.fission_energy_release is not None:
-                fer = openmc.data.FissionEnergyRelease.from_compact_hdf5(
-                    args.fission_energy_release, neutron)
-                if fer is not None:
-                    neutron.fission_energy = fer
-
-            print('Converting {} (ACE) to {} (HDF5)'.format(table.name,
-                                                            neutron.name))
-
-            # Determine filename
-            outfile = os.path.join(args.destination,
-                                   neutron.name.replace('.', '_') + '.h5')
-            neutron.export_to_hdf5(outfile, 'w')
-
-            # Register with library
-            library.register_file(outfile)
-
-        elif table.name.endswith('t'):
+            if name not in nuclides:
+                try:
+                    neutron = openmc.data.IncidentNeutron.from_ace(
+                        table, args.metastable)
+                except Exception as e:
+                    print('Failed to convert {}: {}'.format(table.name, e))
+                    continue
+
+                # Fission energy release data, if available
+                if args.fission_energy_release is not None:
+                    fer = openmc.data.FissionEnergyRelease.from_compact_hdf5(
+                        args.fission_energy_release, neutron)
+                    if fer is not None:
+                        neutron.fission_energy = fer
+
+                print('Converting {} (ACE) to {} (HDF5)'.format(table.name,
+                                                                neutron.name))
+
+                # Determine filename
+                outfile = os.path.join(args.destination,
+                                       neutron.name.replace('.', '_') + '.h5')
+                neutron.export_to_hdf5(outfile, 'w')
+
+                # Register with library
+                library.register_file(outfile)
+
+                # Add nuclide to list
+                nuclides[name] = outfile
+            else:
+                # Then we only need to append the data
+                print('Converting {} (ACE) to {} (HDF5)'.format(table.name,
+                                                                neutron.name))
+                neutron = \
+                    openmc.data.IncidentNeutron.from_hdf5(nuclides[name])
+                neutron.add_temperature_from_ace(table, args.metastable)
+                neutron.export_to_hdf5(outfile + '_1', 'w')
```

Comment:
You're thought process is correct, however I got a surprise from this:
- IncidentNeutron.from_hdf5 opens the hdf5 file and populates an IncidentNeutron object with the data.
- h5py doesnt do a copy of the data as you load it, it just gives you the reference in some cases.  So I can't close the file, or I lose the data (or I add lots of code to do a deep copy)
- So when you do neutron.export_to_hdf5, you cant open the file again (this time to write), so it will fail.  I could have just passed around the file object and originally opened it under mode 'a' but that seemed an odd thing to do from an API standpoint